test/src/test/java/org/corfudb/CorfuTestParameters.java

```diff
@@ -67,6 +67,11 @@
      * cause the (un)expected behavior. */
     public final int CONCURRENCY_LOTS;
 
+    /** Used to indicate when determinstic seeding is to be used
+     */
+    public final long RANDOMIZED_SEED;
```

Comment:
I think we need to just have one variable `RANDOM_SEED`

In the constructor, I would add a new property test.randomSeed, and set it like this:
```
RANDOM_SEED = System.getProperty("test.randomSeed") == null ? 0 : Integer.parse(System.getProperty("test.randomSeed"));
```